packages/landing/src/pages/landing-content.mdx

```diff
@@ -10,15 +10,21 @@ leveraging the language to create expressive & dynamic experiences.
 ```jsx
 /** @jsxImportSource @compiled/react */
 
-export const LargeText = ({ color, children }) => (
-  <span
-    css={{
-      color: color,
-      fontSize: 48,
-    }}>
-    {children}
-  </span>
-);
+import { css } from '@compiled/react';
+
+export const LargeText = ({ color, children }) => {
+  const largeTextStyles = css({
+    fontSize: '48px',
+    color: color,
+  });
+
+  return (
+    <span
+      css={largeTextStyles}>
+      {children}
+    </span>
+  )
+};
```

Comment:
So this forces the runtime element of Compiled, right?  Would perhaps a better example be like this.  Is there value in showing off composition?
```suggestion
const largeTextStyles = css({ fontSize: '48px' });

export const LargeText = ({ color, children }) => {
  const dynamicColorStyles = css({ color: color });

  return (
    <span
      css={[largeTextStyles, dynamicColorStyles]}>
      {children}
    </span>
  )
};
```

Reply:
ooh thats a good idea, ty for the suggestion

Reply:
> Is there value in showing off composition?

to answer this question, it's not strictly necessary as [we have docs](https://compiledcssinjs.com/docs/composition) that cover it

however, i reckon its a common enough use case that we might as well add it yeah